### PR TITLE
Wish list

### PR DESCRIPTION
**Pull request by AnastasijaGurejeva**

### CODE REVIEW

File: app/src/main/java/com/example/mybudget/WishlistActivity.java [hunk 1]
```diff
@@ -46,5 +61,44 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                 return false;
             }
         });
+        initImageBitmaps();
+    }
+
+
+    /*
+     * Method creates a pathway to the other
```
Comment (Beadsley) [inline app/src/main/java/com/example/mybudget/WishlistActivity.java]: change method header

File: app/src/main/java/com/example/mybudget/WishlistActivity.java [hunk 2]
```diff
@@ -46,5 +61,44 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
...
+        mSavingProgress.add(40);
+
+        mImageUrls.add("https://static.thenounproject.com/png/971099-200.png");
+        mWishNames.add("Add your dream");
+        mWishPrices.add(0);
+        mSavingProgress.add(0);
+        initRecyclerView();
+    }
+
+    private void initRecyclerView() {
```
Comment (Beadsley) [inline app/src/main/java/com/example/mybudget/WishlistActivity.java]: needs description

File: app/src/main/java/com/example/mybudget/RecyclerViewAdapter.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    public ViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+
+        View view = LayoutInflater.from(viewGroup.getContext()).inflate(R.layout.layout_wish_list_item, viewGroup, false);
+        ViewHolder viewHolder = new ViewHolder(view);
+        return viewHolder;
+    }
+
+
+    @Override
+    public void onBindViewHolder(@NonNull ViewHolder viewHolder, int i) {
```
Comment (Beadsley) [inline app/src/main/java/com/example/mybudget/RecyclerViewAdapter.java]: description

File: app/src/main/java/com/example/mybudget/WishlistActivity.java (hunk 1 above)
Comment (AnastasijaGurejeva) [reply app/src/main/java/com/example/mybudget/WishlistActivity.java]: Changed

File: app/src/main/java/com/example/mybudget/WishlistActivity.java (hunk 2 above)
Comment (AnastasijaGurejeva) [reply app/src/main/java/com/example/mybudget/WishlistActivity.java]: Resolved

File: app/src/main/java/com/example/mybudget/RecyclerViewAdapter.java (hunk 3 above)
Comment (AnastasijaGurejeva) [reply app/src/main/java/com/example/mybudget/RecyclerViewAdapter.java]: Resolved

**Pull request merged**